CHANGELOG.rst

```diff
@@ -4,7 +4,7 @@ Changelog
 0.5.4 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Delete useless areas and polygons creation/deletion dates
```

Comment:
You'll have to rebase your commit, I did some sh... work to postpone incompatible changes.